packages/grafana-openapi/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "name": "@grafana/openapi",
+  "version": "12.4.0-pre",
+  "description": "Grafana OpenApi Library",
+  "keywords": [
+    "grafana",
+    "openapi",
+    "typescript"
+  ],
+  "type": "module",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "git+http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-openapi"
+  },
+  "exports": {
+    "./api/*": {
+      "import": "./dist/api/*.json",
+      "require": "./dist/api/*.json"
+    },
+    "./apis/*": {
+      "import": "./dist/apis/*.json",
+      "require": "./dist/apis/*.json"
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "./README.md",
+    "./CHANGELOG.md",
+    "LICENSE_APACHE2"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn createfolders && yarn copy-dist",
+    "clean": "rm -rf ./dist && rm -f package.tgz",
+    "copy-api": "cp ../../public/openapi3.json ./src/api",
+    "copy-dist": "cp ./src/apis/* ./dist/apis && cp ./src/api/* ./dist/api",
+    "createfolders": "mkdir dist && mkdir dist/api && mkdir dist/apis",
+    "format": "prettier --list-different \"**/*.json\" --write",
+    "generate:openapi": "make -C ../../ all-openapi3-gen",
+    "typecheck": "",
+    "prepack": "yarn build",
+    "postpack": "",
+    "process-specs": "node src/scripts/process-specs.ts && yarn copy-api && yarn format"
```

Comment:
Do you think we need to format the specs? If yes, maybe we could not use the `--list-different` to reduce the script log output?

Reply:
Fixed prettier in https://github.com/grafana/grafana/pull/114017/commits/9ce9e49da32d57da0d5ea0e3a48ff062fe91d5c0